examples/dt/uni06zeta/control-plane/service-values.yaml

```diff
@@ -116,6 +116,7 @@ data:
       type_drivers = geneve,vlan
       tenant_network_types = geneve,vlan
       extension_drivers = qos,port_security,dns_domain_ports
+      path_mtu = 1400
```

Comment:
let's hold, i think we would need some routes in network attachments. Temporary can get this fix too if route fixes are not yet landed in ci-framework

Reply:
@karelyatin Do we still need to hold on merging this?

Reply:
@abays can we get this in

Reply:
Can we get an ack on whether this has been tested?

Reply:
@abays full testing on zeta job not done, just the affected tests were tested in zeta environment with this config and worked fine, also we have component jobs running with this config already so should be fine.